superset-frontend/src/dashboard/components/SliceHeaderControls/index.tsx

```diff
@@ -310,13 +311,14 @@ class SliceHeaderControls extends React.PureComponent<
 
         {supersetCanShare && (
           <ShareMenuItems
+            dashboardId={dashboardId}
+            hash={componentId}
```

Comment:
```suggestion
            dashboardComponentId={componentId}
```

Can we push down the concept of `hash` to only when we interact with the URL? It took me a while to realize what it really means.

Reply:
That's a good idea, I like it 👍